packages/web/src/server/index.ts

```diff
@@ -98,12 +107,26 @@ export async function track(
 
     const url = new URL(origin);
 
+    let flagsToReport: Record<string, unknown> = {};
+    const allFlags = requestContext?.flags?.getValues();
+    if (options?.flags && allFlags) {
+      options.flags.forEach((key) => {
+        flagsToReport[key] = allFlags[key];
+      });
+    } else if (allFlags) {
+      // Default to all flags. The ingest endpoint will truncate them.
+      flagsToReport = allFlags;
+    }
+
     const body = {
       o: origin,
       ts: new Date().getTime(),
       r: '',
       en: eventName,
       ed: props,
+      flags: {
```

Comment:
should this just be `f` to keep it shorter?

Reply:
Maybe. I think the reason the other keys are so short is a historical one, so I'd rather be more descriptive, but will do whatever the Analytics team sees fit here.